android/src/main/java/in/juspay/hypersdkreact/HyperSdkReactModule.java

```diff
@@ -200,8 +202,43 @@ public void createHyperServices() {
     @ReactMethod(isBlockingSynchronousMethod = true)
     public boolean onBackPressed() {
         synchronized (lock) {
-            return hyperServices != null && hyperServices.onBackPressed();
+            if (isHyperCheckoutLiteInteg){
+                return HyperCheckoutLite.onBackPressed();
+            }else{
+                return hyperServices != null && hyperServices.onBackPressed();
+            }
+        }
+    }
+
+    @ReactMethod
+    public void openPaymentPage(String data) {
+      synchronized (lock) {
+        try {
+          isHyperCheckoutLiteInteg = true;
+          JSONObject sdkPayload = new JSONObject(data);
+          FragmentActivity activity = (FragmentActivity) getCurrentActivity();
+
+          HyperCheckoutLite.openPaymentPage(activity, sdkPayload, new HyperPaymentsCallbackAdapter() {
+            @Override
+            public void onEvent(JSONObject data, JuspayResponseHandler handler) {
+              // Send out the event to the merchant on JS side
+              if (data.optString("event").equals("process_result") && wasProcessWithActivity) {
```

Comment:
@namitgoel - this if block is not needed, since we are not opening a new activity in `openPaymentPage` call